sdk/examples/client/output/build_basic_output.rs

```diff
@@ -1,7 +1,9 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! cargo run --example build_basic_output --release
+//! In this example we will send basic outputs with different feature blocks.
```

Comment:
```suggestion
//! In this example we will build basic outputs with different feature blocks.
```
I know was like this before, but was wrong :trollface: